src/QualificationRunner.Core/Domain/LogWatcher.cs

```diff
@@ -102,7 +96,32 @@ private bool logFileCreated()
 
          private void appendTextToLog()
          {
-            _logger.AddToLog(readFileText(), LogLevel.Debug, null);

```

Comment:
Before I was always adding stuff from the file as debug, which was not great to see what was happening
This is try to find out what level it should be from the  text
This is not bullet proof however but seems to work better so far to create log that can be understood after the fact